tonic-build/src/lib.rs

```diff
@@ -81,6 +81,10 @@ use quote::TokenStreamExt;
 /// Prost generator
 #[cfg(feature = "prost")]
 mod prost;
+#[cfg(feature = "prost")]
+pub use prost_build::Config as ProstBuildConfig;
+#[cfg(feature = "prost")]
+pub use prost_types::FileDescriptorSet;
```

Comment:
Are these already used in tonic-build's public API? Where/how?

Reply:
Given that `Config` is used as `Config` in those functions and given that there is no type definition called `Config` anywhere in the tonic repo, IMO it would be clearer to just import `Config` without aliasing it.